backtrace-library/src/main/java/backtraceio/library/logger/BacktraceLogLogger.java

```diff
@@ -0,0 +1,95 @@
+package backtraceio.library.logger;
+
+import android.util.Log;
+
+/**
+ * Backtrace Logger implementation class for logging messages within the Backtrace library.
+ * <p>
+ * This class leverages the {@link android.util.Log} package to output log messages.
+ * It provides a standardized way to log information, warnings, errors, and debug messages
+ * from within the library, making it easier to track and diagnose issues.
+ * </p>
+ */
+public class BacktraceLogLogger implements Logger {
+
+    private static final String BASE_TAG = "BacktraceLogger: ";
+    /**
+     * Level from which all information is logged
+     */
+    private LogLevel logLevel;
+
+    public BacktraceLogLogger() {
+        this(LogLevel.OFF);
+    }
+
+    public BacktraceLogLogger(LogLevel logLevel) {
+        this.logLevel = logLevel;
```

Comment:
In each logging function you're executing:
`this.logLevel.ordinal()` 

since this is critical path that can be executed quite often, maybe we should set `this.logLevel =logLevel.ordinal();` instead?

Reply:
good point, improved